src/elm/arithmetic.js

```diff
@@ -49,6 +52,8 @@ class Subtract extends Expression {
     const difference = args.reduce((x, y) => {
       if (x.isQuantity || x.isDateTime || x.isDate) {
         return doSubtraction(x, y);
+      } else if (x.isUncertainty && y.isUncertainty) {
+        return new Uncertainty(x.low - y.low, x.high - y.high);
```

Comment:
Actually, this is kind of like the division problem I discussed below.  I don't think it quite works like you've implemented here because something like `Uncertainty(5,10) - Uncertainty(5,10)` is not `Uncertainty(5-5, 10-10)` (which is the same as `0`).  If you think about it logically, we're saying:
> What is _some number between 5 and 10_ minus _some other number between 5 and 10_?

Well, we know the lowest possible result would be if the first number was actually `5` and the second number was actually `10`-- and the highest possible result would be if the first number was actually `10` and the second number was actually `5`.  So I think `Uncertainty(5,10) - Uncertainty(5,10)` should result in `Uncertainty(-5, 5)`, right?  Or more generally, `Uncertainty A - Uncertainty B` is `Uncertainty(A.low - B.high, A.high - B.low)`.

This makes sense if you convert it to addition too.  Another way to think of `- Uncertainty(5, 10)` is `+ Uncertainty(-10, -5)` (but note we had to switch the order of the numbers since `-10` < `-5`).

Anyway, hopefully my babbling makes some amount of sense.